ArmPkg/Library/ArmMmuLib/AArch64/ArmMmuLibPageTableAlloc.c

```diff
@@ -12,7 +12,7 @@
 #include <Library/MemoryAllocationLib.h>
 #include <Protocol/ArmPageTableMemoryAllocation.h>
 
-PAGE_TABLE_MEM_ALLOC_PROTOCOL  *mPageTableMemAllocProtocol = NULL;
+PAGE_TABLE_MEM_ALLOC_PROTOCOL  *mArmMmuPageTableMemAllocProtocol = NULL;

```

Comment:
What does this conflict with? If you change this back, does it still give an error?

Reply:
It conflicts with the variable in 
`armpkg\library\armmmulib\aarch64\armmmulibpagetablealloc.c` and `armpkg\drivers\cpudxe\pagetablememoryallocation.c`:
The error is that here the variable is a pointer type but in those files it's just the struct type.
`Compiler #4744 from  'mPageTableMemAllocProtocol' has different type`

Reply:
These can be made static right, instead of changing the name?

Reply:
That worked, yes. Updated the commit

Reply:
We should continue to use `STATIC`. We only used lowercase `static` for a local variable that is marked static.